### PR TITLE
New Mercurial(hg) Cheat Sheet

### PR DESCRIPTION
**Pull request by ericlake**
**What does your Instant Answer do?**
This instant answer is a cheat sheet for working with the mercurial (hg) dvcs.

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.cheatography.com/codeshane/cheat-sheets/mercurial-hg/

**Why did you choose this data source?**
This was the data source that was linked to in the instant answer idea page (6223)

**Are there any other alternative (better) data sources?**
https://mercurial.selenic.com/wiki/QuickReferenceCardsAndCheatSheets?action=AttachFile&do=view&target=Mercurial-Usage-v1.0bz-300dpi-b%26w.png

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
programmers, sysadmins

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
https://duck.co/ideas/idea/6223/mercurial-cheat-sheet

**Are you having any problems? Do you need our help with anything?**
None that I can think of. However if I am not doing something the way that you would like to see it please let me know and I will do what I can to correct it.

**Where did you hear about DuckDuckHack? (For first time contributors)**
I heard about DuckDuckGo from a coworker. He pointed out that people can contribute to the project through things like instant answers so I started looking into how I could try to help out.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Screenshot provided in comment below.

-----
IA Page: https://duck.co/ia/view/hg_cheat_sheet

### CODE REVIEW

**Comment by ericlake** [comment]
hg_cheat_sheet

**Comment by moollaza** [comment]
@ericlake thanks for your contribution!

Can you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and edit your initial comment. Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

File: share/goodie/cheat_sheets/json/mercurial.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
+{
+    "id": "hg_cheat_sheet",
+    "name": "Mercurial(hg) help",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mercurial.json]: Let's remove "help" from the `name`

File: share/goodie/cheat_sheets/json/mercurial.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    "description": "Mercurial(hg) help",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mercurial.json]: Let's remove this or make it a little more descriptive (or less redundant)

File: share/goodie/cheat_sheets/json/mercurial.json [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    "metadata": {
+        "sourceName": "Mercurial (hg)",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mercurial.json]: Let's change this to "Cheatography"

File: share/goodie/cheat_sheets/json/mercurial.json [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+        "Remote Repositories",
+        "Merge Operations",
+        "Command Options",
+        "Help",
+        "Terminology"
+    ],
+    "sections": {
+        "Create Local Repositories": [{
+            "val": "Creates a ./.hg/ subfolder and repository",
+            "key": "$ hg init \\[project-name\\]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mercurial.json]: Please remove the leading `$` on these. Apologies if you saw another example of this in the repo, we're trying to get rid of any bad examples!

**Comment by moollaza** [comment]
@ericlake thanks for updating the PR template!

I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

**Comment by ericlake** [comment]
@moollaza,

I think that I addressed everything that you mentioned. 

Updated description
removed "help" from "name"
removed "$"s from commands

**Comment by moollaza** [comment]
@ericlake this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

By the way, we think you did a great job with this! Are you interested in helping others who build similar Instant Answers in the future?

If yes, we'll reach out and add you to a private list of mentors as well as send over some DDG merch as an additional thank you. Just email <open@duckduckgo.com> with the subject 'I want to help.' Please include your shipping address and t-shirt size. We'll reach out from there!

Otherwise, thank you for this amazing contribution. We hope you'll keep exploring and playing around with DuckDuckHack.

**Comment by ericlake** [comment]
Thanks for all the help @moollaza. Working on this was quite a bit of fun.

**Comment by moollaza** [comment]
@ericlake congrats! It's live: https://duckduckgo.com/?q=mercurial+cheat+sheet&t=ffcm&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**